low-level/framework/utils/conf_utils.py

```diff
@@ -148,3 +152,8 @@
 
 SRVNODE = Conf.get(GLOBAL_CONF, f'{CLUSTER}>{SERVER_NODES}')[MACHINE_ID]
 ENCLOSURE = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{STORAGE}>{ENCLOSURE_ID}")
+
+SITE_ID = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{'site_id'}",'DC01')
```

Comment:
define default values for all these four in sspl_constants.py and use from there.
DataCenter01
Rack01
ServerNode01
Cluster01

Reply:
Done.